### ISSUE TITLE
Crash with certain invalid path args.

### ISSUE THREAD

**Issue by mitchcapper** [MBulli/SmartCommandlineArgs#148]
Bare paths/args that are invalid (aka will cause File/Path validation commands to throw exceptions) cause hard crashes.  Will submit a PR to fix.

**Comment by Irame** [comment]
I noticed the same issue and think this commit ba064c2442fbe535b7c06b8f221f2b1b6d6d4e74 should fix it.
I should probably catch all exceptions though, to prevent the IDE from crashing.

**Comment by mitchcapper** [comment]
Yeah probably can't hurt to catch them all but you likely have them handled.  I don't *think* it crashes the IDE I think it only crashes the plugi (which then shows as having errors) but I could be wrong.
